bit/subsets.py

```diff
@@ -7,44 +7,54 @@
 For example,
 If nums = [1,2,3], a solution is:
 
-[
-  [3],
-  [1],
-  [2],
-  [1,2,3],
-  [1,3],
-  [2,3],
-  [1,2],
-  []
-]
+{
+    (1, 2),
+    (1, 3),
+    (1,),
+    (2,),
+    (3,),
+    (1, 2, 3),
+    (),
+    (2, 3)
+}
 """
+import unittest
 
 
-def subnets(nums):
-    nums.sort()
-    total = 2 ** len(nums)  # or 1 << len(nums)
-    res = [] * total
+def subsets(nums):
+    """
+    :param nums: List[int]
+    :return: Set[tuple]
+    """
+    # nums.sort()  # no need for sort
```

Comment:
Can we don't remove this line? You commented it out.

Reply:
Yes, you are right. I will remove this.